packages/fxa-auth-server/lib/payments/customer-locations.ts

```diff
@@ -1,77 +1,75 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { BigQuery } from '@google-cloud/bigquery';
+import { PayPalBillingAgreements } from 'fxa-shared/db/models/auth';
+import { uuidTransformer } from 'fxa-shared/db/transformers';
 import { Logger } from 'mozlog';
-
-// Region codes from: https://en.wikipedia.org/wiki/ISO_3166-2:US and https://en.wikipedia.org/wiki/ISO_3166-2:CA
+import Stripe from 'stripe';
+import { StripeHelper, COUNTRIES_LONG_NAME_TO_SHORT_NAME_MAP } from './stripe';
+// Region codes from: https://en.wikipedia.org/wiki/ISO_3166-2:US and https://en.wikipedia.org/wiki/ISO_3166-2:CA with country prefixes removed
 import STATES_LONG_NAME_TO_SHORT_NAME_MAP from './states-long-name-to-short-name-map.json';
-import { StripeHelper } from './stripe';
 
 type PayPalUserLocationResult = {
   uid: string;
-  state: string;
-  country: string;
+  state?: string;
+  country?: string;
   count: number;
 };
 
 const stateNames = STATES_LONG_NAME_TO_SHORT_NAME_MAP as {
   [key: string]: { [key: string]: string };
 };
 
-// The countries we need region data for
-export const COUNTRIES_LONG_NAME_TO_SHORT_NAME_MAP = {
-  // The long name is used in the BigQuery metrics logs; the short name is used
-  // in the Stripe customer billing address
-  'United States': 'US',
-  Canada: 'CA',
-} as { [key: string]: string };
+// GCP credentials are stored in the GOOGLE_APPLICATION_CREDENTIALS env var
+// See https://cloud.google.com/docs/authentication/getting-started
+const GCP_PROJECT_NAME =
+  process.env.FXA_TAX_REPORTING_GCP_PROJECT_NAME || 'bdanforth-fxa-dev';
+const GCP_DATASET_ID =
+  process.env.FXA_TAX_REPORTING_GCP_DATASET_ID || 'fxa_auth_stage';
+const GCP_TABLE_NAME =
+  process.env.FXA_TAX_REPORTING_GCP_TABLE_NAME ||
+  'fxa_4218_uid_state_country_count_01042022';
```

Comment:
This is really odd to have in application code. Configuration values should be looked up from the config object.

Reply:
> This is really odd to have in application code.

Heh can't disagree.

I did confirm with Bianca that she's okay with leaving these values here.  I suppose this type of defaults is not that different from using default values in the convict configs.  We'll remove the defaults.

> Configuration values should be looked up from the config object.

Since this is a one time script we do not plan to add more config values.

Reply:
These are now required options passed into the script command with no defaults. Unfortunately I couldn't get `commander` to enforce the "required" part, so I am throwing errors instead.

TIL: We use `commander` v 2.18 in the auth server, and it's at 9.0.0 at the moment. >.>